example/passthrough_ll.c

```diff
@@ -1240,7 +1240,11 @@ int main(int argc, char *argv[])
 		}
 
 	} else {
-		lo.source = "/";
+		lo.source = strdup("/");
```

Comment:
The true-branch of this if gives us a malloc-ed string.  So for symmetry, we make this one a malloced on, too.  So we can `free` without a care further down.